master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -62,3 +55,9 @@ $ kubectl get node <master_name> -o yaml | grep kubeadm
 * This install does not configure etcd TLS
 * This install expects that one Kubernetes master node has been labeled with `kubeadm.alpha.kubernetes.io/role: master`
 * This install assumes no other pod network has been installed.
+* The CIDR(s) specified with the flag `--cluster-cidr` (pre 1.6) or
+  `--pod-network-cidr` (1.6+) must match the Calico IP Pools to have Network
+  Policy function correctly. The default is `192.168.0.0/16`.
```

Comment:
Ah. The sentence correctyl insinuates its the default values for those flags. I didn't think that was actulaly true. But if it is, no action. Let me give one final look then ill merge